deploy/quickstart/quick_start.py

```diff
@@ -0,0 +1,211 @@
+import os
+from os import path
+import urllib.request
+from subprocess import call
+import subprocess
+
+
+def quick_start():
+    version = "v0.2.0"
+
+    home = os.getenv("HOME")
+    install_dir = request_input_path("Select an installation directory", home + "/jmui")
```

Comment:
nit: use `'{0} is a {1}'.format(variable0, variable1)` or `'{v0} is a {v1}'.format(v0=variable0, v1=variable1)`throughout the script for better readability and performance.